pkg/reloader/reloader.go

```diff
@@ -412,28 +482,87 @@ func hashFile(h hash.Hash, fn string) error {
 	return nil
 }
 
-func (r *Reloader) triggerReload(ctx context.Context) error {
-	req, err := http.NewRequest("POST", r.reloadURL.String(), nil)
+type PIDReloader struct {
+	pname string
+}
+
+func NewPIDReloader(pname string) *PIDReloader {
+	return &PIDReloader{
+		pname: pname,
+	}
+}
+
+func (pr *PIDReloader) TriggerReload(ctx context.Context) error {
+	procs, err := ps.Processes()
```

Comment:
We don't need to be privileged to retrieve PIDs of our own container, right?

Reply:
correct